lib/modules/argonaut/test/argonaut_diagnostic_report_sequence_test.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-require_relative '../../test_helper'
+require_relative '../../../../test/test_helper'
```

Comment:
Is there a better place for `test_helper`?

Reply:
This is ugly, but the tests in `test` and in `lib/modules/some_module/test` both need access to it. I tried to have rake load it, but I couldn't get that to work, perhaps because rake runs in a different process or something. So I'm just leaving it where it is for now.